__DEFINES/subsystem.dm

```diff
@@ -43,9 +43,11 @@
 #define SS_PRIORITY_PLANT          40
 #define SS_PRIORITY_UNSPECIFIED    30
 #define SS_PRIORITY_LIGHTING       20
+#define SS_PRIORITY_AMBIENCE	   19
```

Comment:
This might be something that should be changed? I've admittedly got no clue what the priority of this should be, so I just shoved it under lighting.